lib/spanner/gen_command.ex

```diff
@@ -156,19 +156,14 @@ defmodule Spanner.GenCommand do
 
   def handle_info({:publish, topic, message},
                   %__MODULE__{topic: topic, cb_module: cb_module, bundle_name: bundle}=state) do
-    case Carrier.CredentialManager.verify_signed_message(message) do
-      {true, payload} ->
-        case Command.Request.decode(payload) do
-          {:ok, req} ->
-            process_message(req, cb_module, bundle, state)
-          {:error, error} ->
-            # error is a map; turn it into a string for now to
-            # simplify the error reply contract.
-            {:noreply, send_error_reply(inspect(error), payload["reply_to"], state)}
-        end
-      false ->
-        Logger.error("Message signature not verified! #{inspect message}")
-        {:noreply, state}
+    payload = Poison.decode!(message)
+    case Command.Request.decode(payload) do
+      {:ok, req} ->
+        process_message(req, cb_module, bundle, state)
+      {:error, error} ->
+        # error is a map; turn it into a string for now to
+        # simplify the error reply contract.
+        {:noreply, send_error_reply(inspect(error), payload["reply_to"], state)}
```

Comment:
Might be a good opportunity to fix this up. `send_error_reply/3` doesn't actually modify the state, so it doesn't need to return it. Having a side-effecting operation stuck inside a GenServer return value is rather confusing.

Also, can you elaborate on "simplify the error reply contract"?